Matterhook.NET.MatterhookClient/MattermostAttachment.cs

```diff
@@ -1,51 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using Newtonsoft.Json;
 
 namespace Matterhook.NET.MatterhookClient
-
 {
-    public class MattermostMessage
-    {
-
-        //https://docs.mattermost.com/developer/webhooks-incoming.html
-
-        /// <summary>
-        /// Channel to post to
-        /// </summary>
-        [JsonProperty(PropertyName = "channel")]
-        public string Channel { get; set; }
-
-        /// <summary>
-        /// Username for bot
-        /// </summary>
-        [JsonProperty(PropertyName = "username")]
-        public string Username { get; set; }
-
-        /// <summary>
-        /// Bot/User Icon
-        /// </summary>
-        [JsonProperty(PropertyName = "icon_url")]
-        public string IconUrl { get; set; }
-
-        /// <summary>
-        /// Message body. Supports Markdown
-        /// </summary>
-        [JsonProperty(PropertyName = "text")]
-        public string Text { get; set; }
-
-        /// <summary>
-        /// Richtext attachments 
-        /// </summary>
-        [JsonProperty(PropertyName = "attachments")]
-        public List<MattermostAttachment> Attachments { get; set; }
-
-    }
-
     /// <summary>
     /// https://docs.mattermost.com/developer/message-attachments.html#message-attachments
     /// </summary>
-    public class MattermostAttachment
+    public class MattermostAttachment/*ICloneable<MattermostAttachment>*/
```

Comment:
Is there a reason for commenting out the inheritance here? I'm guessing you planned for an interface but ended up not implementing?

```suggestion
    public class MattermostAttachment
```

Reply:
Yes I first thought & wrote an interface extending `ICloneable`. Didn't use it at the end, since I didn't see any use cases, but we still could implement it if wished,

Reply:
For the sake of readability, I'd rather leave out commented code that isn't doing anything :)